src/hardware/design_parameters.rs

```diff
@@ -9,6 +9,7 @@ pub const ADC_DAC_SCK_MAX: MegaHertz = MegaHertz(50);
 
 /// The optimal counting frequency of the hardware timers used for timestamping and sampling.
 pub const TIMER_FREQUENCY: MegaHertz = MegaHertz(100);
+pub const TIMER_PERIOD: f32 = 1. / ((TIMER_FREQUENCY.0 * 1_000_000) as f32);
```

Comment:
Minor nit, but could we just do like `TIMER_FREQUENCY.into<Herz>().0 as f32` so we don't have to carry around the 1_000_000 constant? The math would break if the type of `TIMER_FREQUENCY` changed

Reply:
That's unfortunately not const.